saleor/core/otel/metrics.py

```diff
@@ -0,0 +1,52 @@
+from typing import cast
+
+from opentelemetry import metrics
+from opentelemetry.metrics import Meter
+from opentelemetry.util.types import Attributes
+
+from ... import __version__
+from .context import enrich_with_trace_attributes
+
+
+class _ContextAwareMeter(Meter):
```

Comment:
This class inherits from `Meter` but does not use that. It's a facade, not a subclass.